addons/point_of_sale/controllers/main.py

```diff
@@ -220,7 +220,7 @@ def _parse_additional_values(fields, prefix, kwargs):
                 form_values['partner_address'] = partner._display_address()
 
         return request.render("point_of_sale.ticket_validation_screen", {
-            'partner': partner,
+            'partner_sudo': partner,
```

Comment:
Here it seems like it's inconsistent whether it's sudo or not, request is not sudo but pos_order is.